requirements.txt

```diff
@@ -1,3 +1,4 @@
-sphinx==4.0.2
-sphinx_rtd_theme==0.5.2
+sphinx
```

Comment:
We have pins on sphinx and sphinx-autobuild, but I honestly don't know why. I'm assuming this was the case here?

Reply:
This introduces sphinx-autobuild for the first time "officially" on the research side, though I have been using it since 2022. The pin to 4.0.2 was probably pretty current at the time it was made, but I don't know why it was pinned.